### PR TITLE
[SPV -> SPV-IR] Fix image builtin mangling of unsigned type

### PR DESCRIPTION
**Pull request by wenju-he**
Return type of image read and Texel type of image write builtins may be
unsigned. Before this PR, the builtin names in SPIR-V Friendly IR were
always mangled with signed type.

### CODE REVIEW

File: lib/SPIRV/SPIRVInternal.h [hunk 1]
```diff
@@ -921,7 +921,8 @@ std::string getSPIRVFriendlyIRFunctionName(OCLExtOpKind ExtOpId,
 /// \param Types of arguments of SPIR-V built-in function
 /// \return IA64 mangled name.
 std::string getSPIRVFriendlyIRFunctionName(const std::string &UniqName,
-                                           spv::Op OC, ArrayRef<Type *> ArgTys);
+                                           spv::Op OC, ArrayRef<Type *> ArgTys,
+                                           ArrayRef<SPIRVValue *> Ops);
```
Comment (vmaksimo) [inline lib/SPIRV/SPIRVInternal.h:925]: Please update the comment above

**Review by vmaksimo — COMMENTED** [review]
LGTM, just one comment

File: lib/SPIRV/SPIRVInternal.h (hunk 1 above)
Comment (wenju-he) [reply lib/SPIRV/SPIRVInternal.h:926]: done, also renamed C to ImOpMask

**Comment by wenju-he** [comment]
See https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2274 for clang-tidy and out-of-tree build error

File: lib/SPIRV/SPIRVInternal.h [hunk 2]
```diff
@@ -919,9 +922,11 @@ std::string getSPIRVFriendlyIRFunctionName(OCLExtOpKind ExtOpId,
 /// \param OC opcode of corresponding built-in instruction. Used to gather info
 /// for unsigned/constant arguments.
 /// \param Types of arguments of SPIR-V built-in function
+/// \param Operands of SPIRVInstruction
```
Comment (svenvh) [inline lib/SPIRV/SPIRVInternal.h:925]:
```suggestion
/// \param Ops Operands of SPIRVInstruction
```
(I'm aware the previous line isn't using `\param` correctly either, but that's outside of this PR's scope.)

Comment (wenju-he) [reply lib/SPIRV/SPIRVInternal.h]: done

**Review by svenvh — APPROVED** [review]

**Review by MrSidims — APPROVED** [review]

**Pull request merged**